### PR TITLE
Update linux-bootsrap-2

### PR DESCRIPTION
**Pull request by Waqar144**
Some more cleanups for GDT and protected mode. Added more explanations.
Some sentence structure corrections.

### CODE REVIEW

File: Booting/linux-bootstrap-2.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Kernel booting process. Part 2.
 ================================================================================
 
-First steps in the kernel setup
+Overview
```
Comment (0xAX) [inline Booting/linux-bootstrap-2.md]: Return first variant please here

File: Booting/linux-bootstrap-2.md [hunk 2]
```diff
@@ -20,16 +20,18 @@ Protected mode
...
-Memory management in protected mode is divided into two, almost independent parts:
+Protected mode brought many changes, but the main one is the difference in memory management. The 20-bit address bus was replaced with a 32-bit address bus. It allowed access to 4 Gigabytes of memory vs 1 Megabyte of real mode. Also [paging](http://en.wikipedia.org/wiki/Paging) support was added, which you can read about in the next sections.
+
+Memory management in Protected mode is divided into two, almost independent parts:
 
 * Segmentation
 * Paging
 
-Here we will only see segmentation. Paging will be discussed in the next sections. 
+Here we will only see Segmentation. Paging will be discussed in the next sections.
```
Comment (0xAX) [inline Booting/linux-bootstrap-2.md]: Why this change?

**Comment by 0xAX** [comment]
Hello @Waqar144 thank you for the PR, I've left some comments.

File: Booting/linux-bootstrap-2.md (hunk 1 above)
Comment (Waqar144) [inline Booting/linux-bootstrap-2.md]: Ok, but I still think this section is more of an overiew.

File: Booting/linux-bootstrap-2.md (hunk 2 above)
Comment (Waqar144) [inline Booting/linux-bootstrap-2.md]: My mistake,
